### PR TITLE
Feat(#19): add table UI

### LINKED ISSUE #19: Monitored Elements List component

### PR DESCRIPTION
**Pull request by mari1912**
## Description

- Add table ui
- add table component
- todo: add i18n

## Related Issues
- Closes #19

### CODE REVIEW

File: dashboard/tailwind.config.js [hunk 1]
```diff
@@ -33,10 +33,14 @@ module.exports = {
...
-        'onSecondary' : '#FFFFFF',
         'lightBlue': '#11B3E6', 
         'onSecondary-10': '#FFFFFF1A',
-        'dark': '#000'
+        'dark': '#000',
+        "white": '#FFFFFF',
+        "lightGray": '#F4F4F4',
+        "mediumGray": '#EAEAEA',
+        "darkGray": '#D6D6D6',
+        "black": '#000000'
```
Comment (rfgvieira) [inline dashboard/tailwind.config.js:43]: this color already exists, `dark` color has the same value as `black`

File: dashboard/src/components/LateralMenu/LateralMenu.tsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ const LateralMenu = (): JSX.Element => {
       orientation="vertical"
     >
       <div className="w-full px-4">
-        <img src="./src/assets/kernelCI_logo.png"className="size-14 text-onSecondary" />
+        <img src="./src/assets/kernelCI_logo.png"className="size-14 text-white" />
```
Comment (lfjnascimento) [inline dashboard/src/components/LateralMenu/LateralMenu.tsx:74]: why does an `img` have a text style?

File: dashboard/src/components/Table/TreeTable.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      ))
+    );
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [treeTableRows]);
+
+  return(
+    BaseTable({
+      headers: treeTableColumns,
+      body: <>{treeTableBody}</>
+    })
```
Comment (lfjnascimento) [inline dashboard/src/components/Table/TreeTable.tsx:93]: why are you using BaseTable as a simple function and not as a component?

File: dashboard/src/components/ui/table.tsx [hunk 4]
```diff
@@ -20,7 +20,7 @@ const TableHeader = React.forwardRef<
   HTMLTableSectionElement,
   React.HTMLAttributes<HTMLTableSectionElement>
 >(({ className, ...props }, ref) => (
-  <thead ref={ref} className={cn("[&_tr]:border-b", className)} {...props} />
+  <thead ref={ref} className={cn("[&_tr]:border-b border-darkGray", className)} {...props} />
```
Comment (lfjnascimento) [inline dashboard/src/components/ui/table.tsx:23]: Let's try not to modify the shadcn components, we can pass the style with the `className` prop

File: dashboard/tailwind.config.js (hunk 1 above)
Comment (lfjnascimento) [reply dashboard/tailwind.config.js:42]: could you separate this change from this commit?

**Review by lfjnascimento — APPROVED** [review]

**Review by rfgvieira — APPROVED** [review]
nice

**Pull request merged**